ChangeLog.md

```diff
@@ -3,6 +3,309 @@
+
+
+### Detailed Changelog
+
+
+#### Graphics / GUI
+
+- New default font Roboto ver. 2.001101 (2014) (with a few extra
+missing glyphs needed used in the GUI)
+- Fixed layout of production / research info panel
+- Reworked rendering of various GUI windows to improve performance
+- Improved functionality of pedia search box
+ - Get a list of results instead of jumping to the first matching entry found
+- Added mousewheel cycling through items in droplists
+- Improved window layout in response to game window resizing
+- Made queue and info windows resizable / movable
+- Increased default tooltip delay
+- Increased default sitrep icon size
+- Added a tooltip for when the in-game save button is disabled.
+- Added messages when game save is started.
+- Improved blending between empires' visibility radii colours
+- Added various hotkey commands, including mapping the map or tech view
+- Suppressed rendering of ridiculously huge visibility circles, such
+as from super testers
+- Added menu button to open freeorion.org
+- Tweaked appearance of colour picker indicator in HS square to have a
+hole/outline where the lines meet
+- Add the distance scale circle to the MapWnd right-click menu
+- Added separate option for FPS limit when the game window doesn't
+have OS input focus
+- Added 3D starfield effect on map
+- Add an entry on the right-click menu to add to top of build queue
+- When left-clicking production queue items, open the system and
+select the planet instead of just jumping to the system on the map.
+- Made ctrl-left click remove an item from build queue.
+- Made ctrl-double click add entries to the top of the build queue
+- Pedia articles can now have images in them
+- New icons for auto and manual turn cycling button
+- Tweaked Sidepanel planet pole tilt angle
+- Tweaked Sitrep layout
+- Reworked rotating selection indicator rendering to be smoother
+- Re-enabled sitreps for winning / losing the game
+ - Add an icon to the PlayerListWnd when an empire wins/loses
+- Made Pedia ship design detail page show troop stats reflecting
+racial bonus/malus
+- Clicking a sitrep planet handle will select the planet for production
+- Tweaks to rendering techs on tech screen when zooming
+- Positioned cursor at end of historical input from pressing up/down
+in messages window
+- Added current turn expected progress to research and production
+progress meters
+- Added specialized tooltip for resource icons
+- Made encyclopedia detail panel of production and research screens closable
+- Switched a few objects list columns from showing object ids to object names
+- Added nearest system column and number of specials column to objects list
+- Production and research queue items show max spendable PP or RP per turn
+- Added ship rally point tracking / setting to production queue. New
+fleets are grouped by rally point on the turn they are produced. Ship
+production items can be right clicked to set their rally point to the
+selected system.
```

Comment:
This should be in Key Changes, or at least a summary of it, it's very useful and has been requested several times.